packages/core-utils/src/__tests__/route.js

```diff
@@ -135,4 +136,60 @@ describe("util > route", () => {
       expect(sortRoutes(route12, route13)).toMatchSnapshot();
     });
   });
+
+  describe("route text color generation", () => {
+    it("should not modify appropriate color pairings", () => {
+      expect(getMostReadableTextColor("#EE352E", "#ffffff")).toBe("#ffffff");
+      expect(getMostReadableTextColor("#00933C", "#ffffff")).toBe("#ffffff");
+      expect(getMostReadableTextColor("#b933ad", "#ffffff")).toBe("#ffffff");
+      expect(getMostReadableTextColor("#808183", "#ffffff")).toBe("#ffffff");
+      expect(getMostReadableTextColor("#0039a6", "#ffffff")).toBe("#ffffff");
+
+      // Both work
```

Comment:
Maybe say instead that the function is indifferent with respect to the proposed text color.

Reply:
Comment expanded